.github/pull_request_template.md

```diff
@@ -1,6 +1,3 @@
 Please make sure to document all user-facing changes in the `CANGELOG.md` file.
```

Comment:
I know this is already merged but...

`CANGELOG.md` -> `CHANGELOG.md`

Let's see how many times we can "cange" the GitHub templates :P

Reply:
@chrisduerr  Can we ninja update it in #3051 :thinking:  ?

Reply:
I like that idea.

Reply:
Nice commit message.

Reply:
LOL, never rely on me to catch spelling issues. I can barely spell my own name.